docs/openj9_support.md

```diff
@@ -72,26 +71,93 @@ show the minimum build configurations that we have tested to build an OpenJDK bi
 **NOTE:** If you obtain pre-built binaries from [AdoptOpenJDK.net](https://adoptopenjdk.net/index.html),
 platform support might vary, depending on their build environment.
 
-### OpenJDK 8 build environment
+### OpenJDK 8
+
+OpenJDK8 binaries are supported on the minimum operating system levels shown in the following tables:
+
+
+| Linux                                 |   x32  |  x64   | ppc64le    | Z31  | Z64 |
+|---------------------------------------|--------|--------|------------|------|------|
+| Centos 6                              |   Y    |   Y    |     Y      |   N  |  N   |
+| Centos 7                              |   Y    |   Y    |     Y      |   N  |  N   |
+| Red Hat Enterprise Linux (RHEL) 6     |   Y    |   Y    |     Y      |   Y  |  Y   |
+| RHEL 7                                |   Y    |   Y    |     Y      |   Y  |  Y   |
+| SUSE Linux Enterprise Server (SLES) 12|   Y    |   Y    |     Y      |   Y  |  Y   |
+| Ubuntu 16.04                          |   Y    |   Y    |     Y      |   N  |  Y   |
+| Ubuntu 18.04                          |   Y    |   Y    |     Y      |   N  |  Y   |
+
+<i class="fa fa-pencil-square-o" aria-hidden="true"></i><span class="sr-only">Note</span> **Note:** Not all of these distributions are tested, but Linux distributions that have a
+minimum glibc version 2.12 are expected to function without problems.
+
+| Windows                    |  x32   |  x64  |
+|----------------------------|--------|-------|
+| Windows 10                 |   Y    |   Y   |
+| Windows Server 2012        |   N    |   Y   |
+| Windows Server 2012 R2     |   N    |   Y   |
+| Windows Server 2016        |   N    |   Y   |
+
```

Comment:
Why the limitation of 32-bit on Servers OSes? Its a bit weird we use Server to build 32-bit but its not supported to run there.

Reply:
That was something I was going to ask you about. Fixed my guess.